src/main/java/org/jenkinsci/plugins/parallel_test_executor/SplitStep.java

```diff
@@ -65,10 +76,12 @@ public DescriptorImpl() {
 
         @Override protected List<?> run() throws Exception {
             if (step.generateInclusions) {
-                return ParallelTestExecutor.findTestSplits(step.parallelism, build, listener, step.generateInclusions);
+                return ParallelTestExecutor.findTestSplits(step.parallelism, build, listener, step.generateInclusions,
+                        new PreviousTestResultLookup.LookupInStage(step.stage));
```

Comment:
This bypasses the `requirePlugins` and will simply throw `NoClassDefFoundError`s if some of those plugins are missing.

Reply:
Ah, yeah, good point. Will fix.